packages/host/app/components/operator-mode/profile-info-popover.gts

```diff
@@ -22,68 +28,202 @@ interface ProfileInfoSignature {
 }
 
 export default class ProfileInfoPopover extends Component<ProfileInfoPopoverSignature> {
-  @service declare matrixService: MatrixService;
-
-  @action logout() {
-    this.matrixService.logout();
-  }
-
   <template>
     <style scoped>
       .profile-popover {
-        width: 280px;
-        height: 280px;
+        width: 300px;
         position: absolute;
         bottom: 68px;
         left: 20px;
         z-index: 3;
-        background: white;
+        background: var(--boxel-100);
         padding: var(--boxel-sp);
         flex-direction: column;
         border-radius: var(--boxel-border-radius);
         box-shadow: var(--boxel-deep-box-shadow);
         display: flex;
       }
+      :deep(.profile-popover-body) {
+        padding: var(--boxel-sp-xl) 0;
+      }
 
       .header {
         display: flex;
         justify-content: space-between;
         align-items: center;
       }
 
-      .header .label {
+      .header button {
+        --boxel-button-font: 600 var(--boxel-font-xs);
+      }
+
+      .credit-info {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        flex-wrap: wrap;
+        gap: var(--boxel-sp-lg);
+        margin-bottom: var(--boxel-sp);
+        padding-top: var(--boxel-sp-lg);
+        border-top: 1px solid var(--boxel-dark);
+      }
+      .info-group {
+        display: flex;
+        flex-direction: column;
+        gap: var(--boxel-sp-xxs);
+      }
+      .label {
+        color: var(--boxel-dark);
+        font: var(--boxel-font-xs);
+      }
+      .out-of-credit {
+        font: var(--boxel-font-sm);
+      }
+      .info-group .value {
         color: var(--boxel-dark);
-        text-transform: uppercase;
+        font: 700 var(--boxel-font-sm);
+        font-style: capitalize;
       }
     </style>
 
     <div class='profile-popover' data-test-profile-popover ...attributes>
       <header class='header'>
-        <div class='label'>
-          Signed in as
-        </div>
-
         <BoxelButton
           @kind='secondary-light'
-          @size='extra-small'
+          @size='small'
           {{on 'click' @toggleProfileSettings}}
           data-test-settings-button
         >
           Settings
         </BoxelButton>
+
+        <BoxelButton
+          @kind='primary-dark'
+          @size='small'
+          {{on 'click' this.logout}}
+          data-test-signout-button
+        >
+          Sign Out
+        </BoxelButton>
       </header>
 
       <ProfileInfo />
-
-      <BoxelButton
-        {{on 'click' this.logout}}
-        @kind='primary-dark'
-        data-test-signout-button
-      >
-        Sign out
-      </BoxelButton>
+      {{! TODO: Remove config.APP.creditInfoEnabled once the API integration for credit info is completed. }}
```

Comment:
Would be better to name this in a more general way, such as "stripeBillingFeatureEnabled", because I think other frontend PRs related to stripe feature will also use this until the backend is ready